src/Adyen/Util/ManualCapture.php

```diff
@@ -0,0 +1,108 @@
+<?php
+/**
+ *                       ######
+ *                       ######
+ * ############    ####( ######  #####. ######  ############   ############
+ * #############  #####( ######  #####. ######  #############  #############
+ *        ######  #####( ######  #####. ######  #####  ######  #####  ######
+ * ###### ######  #####( ######  #####. ######  #####  #####   #####  ######
+ * ###### ######  #####( ######  #####. ######  #####          #####  ######
+ * #############  #############  #############  #############  #####  ######
+ *  ############   ############  #############   ############  #####  ######
+ *                                      ######
+ *                               #############
+ *                               ############
+ *
+ * Adyen API Library for PHP
+ *
+ * Copyright (c) 2022 Adyen N.V.
+ * This file is open source and available under the MIT license.
+ * See the LICENSE file for more info.
+ *
+ */
+
+namespace Adyen\Util;
+
+class ManualCapture
+{
+    private $openInvoice;
+
+    /**
+     * ManualCapture constructor.
+     */
+    public function __construct()
+    {
+        $this->openInvoice = new OpenInvoice();
+    }
+
+    public function isManualCaptureSupported($notificationPaymentMethod): bool
+    {
+        $manualCaptureAllowed = false;
+        // For all openinvoice methods manual capture is the default
+        if ($this->openInvoice->isOpenInvoicePaymentMethod($notificationPaymentMethod)) {
+            return true;
+        }
+
+        switch ($notificationPaymentMethod) {
```

Comment:
I think it would be better to define a static array of constants for these methods